lib/gates_cirq.h

```diff
@@ -285,8 +309,10 @@ struct CXPowGate {
   }
 };
 
-// The (exponent=phi/pi, global_shift=-0.5) instance of XPowGate.
-// This is a function in Cirq.
+/**
```

Comment:
It might be worth adding some additional info for many gates (H, S, T, CZ, CZ, X, Y, Z, XX, YY, ZZ, SWAP, ISWAP, rx, ry, rz, and perhaps PhasedXPowGate, PhasedXZGate, PhasedISwapPowGate, riswap, givens) as in the Cirq API reference. For instance, "The Hadamard gate." can be added to the description of the H gate.

Reply:
I will fill out the additional detail here and merge this PR prior to cutting the new version. This should be ready later today.

Reply:
I've added some detail to each of these, but tried to keep them concise (e.g. no matrix definitions for gates).